lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
```

Comment:
Clear classes for each process, compliant with separation of concerns, ease of debugging and simplicity